modules/visual_script/visual_script_editor.cpp

```diff
@@ -4756,20 +4756,20 @@ VisualScriptEditor::VisualScriptEditor() {
 	HBoxContainer *graph_hbc = graph->get_zoom_hbox();
 
 	Label *base_lbl = memnew(Label);
-	base_lbl->set_text("Change Base Type: ");
+	base_lbl->set_text(TTR("Change Base Type: "));
```

Comment:
The trailing space should not be part of the translatable string.

Reply:
There are many other existing `TTR` label text with a trailing space. For example:

https://github.com/godotengine/godot/blob/7f62ee69b1e6e85803e3b48f0d760f77587f2db9/editor/find_in_files.cpp#L518

I can think of one use case of a translatable trailing space: CJK colon mark doesn't need a space following it.

Reply:
These are exceptions more than the rule, I've fixed tons of occurrences to ensure that there is no space after `:` in translatable strings.

It would be worth reconsidering for CJK, but it adds a major pain for all other languages who will have to ensure not to delete this trailing space (I've seen countless translations in other projects where this situation led to "text:like this").

Reply:
```
$ rg 'TTR.*: "\)' | wc -l
17
$ rg 'TTR.*:"\)' | wc -l
325
```

This shouldn't be changed now though (apart from this new translation), as it would mark all affected translations as fuzzy, and it's too late in the release cycle now to ask all translators to handle those in time for the release. We can fix the 17 occurrences after the 3.2 release, and discuss if we want to change the policy to include trailing space in strings ending with a colon (or figure out a better system to handle it).

Reply:
OK. I'll make the change.

I also found that trailing spaces in `"VCenter Wide "` and `"HCenter Wide "` here seems redundant. Should I just remove the space?

https://github.com/godotengine/godot/blob/5ed57aea96a5016979073b9b24d311ae141516bd/editor/plugins/canvas_item_editor_plugin.cpp#L3862-L3863

Reply:
Ah yes, please do.